### PR TITLE
feat: added ability to pass a custom message

### PR DESCRIPTION
**Pull request by Aghassi**
Currently, this warning just display line that says "You are outdated, please update". However, a user may not know what to do if this is their first time using your CLI. It may be useful to provide the user some context on how to update. 

For example, we use `brew` as our install method of choice. A user may assume they should run `brew upgrade` to get the latest (not the case). To make it more obvious, we can hint to the user by saying "Please run my-cli update to update". This change adds that ability.

### CODE REVIEW

**Comment by jdxcode** [comment]
this is a good change, but I would prefer we use a lodash template so it can be overridden entirely instead of just appended to

**Comment by Aghassi** [comment]
@jdxcode Can you provide me an example of what you mean by "lodash template"? I'm happy to update the PR once I understand 😄

**Comment by Aghassi** [comment]
@jdxcode Ok I refactored it. I _think_ that is what you meant. Also, this is my first jump into TypeScript, so if I messed the syntax up I apologize.

**Comment by Aghassi** [comment]
@jdxcode Does this repo support any CI for PRs? I want to make sure that my changes aren't going to break anything. I've run `yarn test` and `yarn lint` locally by hand to validate changes, but I wanted to know if I should do anything else.

Also, would you be opposed to me adding [husky](https://github.com/typicode/husky) in a separate PR so that certain checks running before commit and push locally?

File: README.md [hunk 1]
```diff
@@ -61,6 +64,7 @@ any of the following configuration properties:
     ],
     "warn-if-update-available": {
       "timeoutInDays": 7,
+      "message": "Run @scope/my-cli update to update to the latest"
```
[the remark this answers is not in the document]
Comment (Aghassi) [reply README.md]: Running `yarn lint` locally passes 😞

**Comment by Aghassi** [comment]
@jdxcode Anything else you need on my end to get this merged in?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (krohrsb) [reply README.md]: It's just in the readme :)

**Comment by Aghassi** [comment]
Well I guess it wouldn’t get caught lol. Will fix.

--
~David Aghassi

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com/
```
Comment (jdxcode) [inline .npmrc]: can this be removed?

File: package.json [hunk 3]
```diff
@@ -8,10 +8,12 @@
     "@oclif/command": "^1.5.3",
     "@oclif/config": "^1.8.7",
     "@oclif/errors": "^1.2.2",
+    "@types/lodash.template": "4.4.4",
```
Comment (jdxcode) [inline package.json]: this should be a dev dependency

File: src/hooks/init/check-update.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ import {Hook} from '@oclif/config'
 import Chalk from 'chalk'
 import {spawn} from 'child_process'
 import * as fs from 'fs-extra'
+import template = require('lodash.template')
```
Comment (jdxcode) [inline src/hooks/init/check-update.ts]: this import could be moved down lower

File: .npmrc (hunk 2 above)
Comment (Aghassi) [reply .npmrc]: Yup. I was trying to handle work from my work laptop and it was using our internal registry. I will remove this.

**Comment by Aghassi** [comment]
@jdxcode Addressed comments

**Pull request merged**